x/oracle/keeper/grpc_query.go

```diff
@@ -0,0 +1,204 @@
+package keeper
+
+import (
+	"context"
+
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/umee-network/umee/x/oracle/types"
+)
+
+// querier is used as Keeper will have duplicate methods if used directly, and gRPC names take precedence over q
```

Comment:
Did this comment get cut off?

Reply:
q = querier, this was more about nomenclature than anything, I'll just update it to describe the actual structure